_targets/meta/meta

```diff
@@ -35,6 +35,7 @@ hv_token|stem|059b668fd0effe34|268e267fa1b9be68|d19cbbef15d7e0e2|268012145||t197
 incoming_data_collated_csvs|stem|526cc5b8a51c611b|c67bb54d6480df1d|46f52353b4288e52|-1436724047||t19703.8407897502s|e1a91a3d0391642b|11588|rds|local|vector|||0.264||
 incoming_data_csvs|stem|0512ae32462262bc|f161ac32f776b1e2|d09e3fc86eeb9554|1255066382||t19689.8810172193s|63e5c65d6d1ce375|134|rds|local|vector|||3656.84||
 incoming_data_csvs_ab32d911|branch||6a31d92726790c7b|c2829ed4f3a9873e|1879218812||t19678.8947293788s||0|rds|local|vector|incoming_data_csvs||0.021||no applicable method for mutate applied to an object of class tryerror
```

Comment:
Q: Should this meta file be in the repo since you ignored the folder earlier?

Reply:
yes... but since it was already in the repo I will do some fenagling to make it act right. (probably this https://stackoverflow.com/questions/1139762/ignore-files-that-have-already-been-committed-to-a-git-repository)